src/services/peerconnection.ts

```diff
@@ -285,6 +300,13 @@ export class PeerConnectionHelper {
      * Unbind all event handler and abruptly close the peer connection.
      */
     public close(): void {
+        // Cancel connection failed timer
+        if (this.connectionFailedTimer !== null) {
+            this.timeoutService.cancel(this.connectionFailedTimer);
+            this.connectionFailedTimer = null;
```

Comment:
When you call `close()` from [here](https://github.com/threema-ch/threema-web/pull/969/files#diff-f0ddf3d90ae1f0bfd272dc1a4e352307R158), the timer will always have been set to null (without cancelling it?). But maybe we should still keep the code above, just to make sure that the timeout is cancelled?

Reply:
Yup, since it can be called from the outside.